src/platforms/woocommerce/index.ts

```diff
@@ -0,0 +1,22 @@
+import { client } from '@/platforms/woocommerce/client';
+import { icon } from '@/platforms/woocommerce/icon';
+import { auth, platform } from '@/sdk';
+import * as constants from './constants';
+
+export default platform('woocommerce', {
+  auth: auth.apiToken({
+    questions: [
+      { type: 'text', id: 'wooCommerceUrl', label: 'WooCommerce URL' },
```

Comment:
I guess we'll come back and write a guide for how to find these if anyone asks hahah